db/migrate/20220920172912_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table(:users) do |t|
+      ## Required
+      t.string :provider, null: false, default: 'email'
+      t.string :uid, null: false, default: ''
+
+      ## Database authenticatable
+      t.string :encrypted_password, null: false, default: ''
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+      t.boolean  :allow_password_change, default: false
+
+      ## Confirmable
+      t.string   :confirmation_token
+      t.datetime :confirmed_at
+      t.datetime :confirmation_sent_at
+      t.string   :unconfirmed_email # Only if using reconfirmable
```

Comment:
Do we need these fields? This Devise module is not enabled in the `User` model

Reply:
There is a ticket to [confirm the account] after the user is created. (https://trello.com/c/VJUN2VuO/16-3-as-a-user-i-should-be-able-to-confirm-my-account-after-signing-up-in-order-to-ensure-that-all-the-users-email-are-valid)

Reply:
Ok so then please enable that module in the model, so migrations and the setup are consistent

Reply:
@JulianPasquale WDYTA implementing those changes in a new PR when that ticket is in progress.
Adicional tests will be required for that too!

Reply:
I don't think any additional test is needed. Just to clarify, the only change you'd have to do is adding `:confirmable` to the `devise` method in the User model
```ruby
devise :database_authenticatable, :registerable,
       :recoverable, :validatable, :confirmable
```

But yeah, I'm ok with whatever you decide to do 🙂

Reply:
User model updated!